assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/controllers.tf

```diff
@@ -13,6 +13,70 @@ resource "aws_route53_record" "etcds" {
   records = [aws_instance.controllers[count.index].private_ip]
 }
 
+# IAM Policy
+resource "aws_iam_instance_profile" "csi-driver-instance-profile" {
+  count = var.set_csi_driver_iam_role ? 1 : 0
+  name  = "${var.cluster_name}-iprof"
+  role  = join("", aws_iam_role.csi-driver-role.*.name)
+}
+
+resource "aws_iam_role_policy" "csi-driver-role-policy" {
+  count = var.set_csi_driver_iam_role ? 1 : 0
+  name  = "${var.cluster_name}-policy"
```

Comment:
You don't need the cluster name in the policy name. This is an *inline* policy, meaning it is created under a specific IAM role and only applies to that role. You shouldn't have name clashes when deploying multiple clusters.